extensions/shared/external-media/sources/with-media.js

```diff
@@ -69,7 +69,7 @@ export default function withMedia() {
 				 * https://github.com/WordPress/gutenberg/issues/22940 is fixed.
 				 */
 				if ( [ UP, DOWN, LEFT, RIGHT ].includes( event.keyCode ) ) {
-					event.stopPropagation();
+					event.preventDefault();
```

Comment:
I'm afraid we can't use `preventDefault` here - it blocks the navigation inside i.e. the Pexels search input as it's one of the default `keydown` events.

Reply:
In which case we should probably add a comment to help future devs to be aware of this.

Reply:
Ugh.  I knew it was too good to be true.

Reply:
Reverted in https://github.com/Automattic/jetpack/pull/16118/commits/0971fcf24dc015c6d143d36924856b6d3f06546c